main.js

```diff
@@ -0,0 +1,136 @@
+var currentWeatherData  = [];
+var fiveDayForecastData = [];
+
+var fetchWeatherData = function(city) {
+  $.ajax({
+    method: "GET",
+    url: `http://api.openweathermap.org/data/2.5/weather?q=${city}&units=imperial&appid=83b3a3ff4100ab8f87dc416f724b6120`,
+    dataType: "json",
+    success: function(data) {
+      addCurrentWeatherData(data);
+    },
+    error: function(jqXHR, textStatus, errorThrown) {
+      console.log(textStatus);
+      alert(errorThrown);
+    }
+  });
+  $.ajax({
+    method: "GET",
+    url: `http://api.openweathermap.org/data/2.5/forecast?q=${city}&units=imperial&appid=83b3a3ff4100ab8f87dc416f724b6120`,
+    dataType: "json",
+    success: function(data) {
+      addFiveDayForecastData(data);
+    },
+    error: function(jqXHR, textStatus, errorThrown) {
+      console.log(textStatus);
+    }
+  });
+};
+
+var addCurrentWeatherData = function(data) {
+  currentWeatherData  = [];
+  
+  var city = data.name;
+  var temp = Math.round(data.main.temp) + "\u00B0";
+  var descrip = data.weather[0].main;
+  var iconUrl = `http://openweathermap.org/img/wn/${data.weather[0].icon}@2x.png`;
+  
+  var pushObj = {
+    temp: temp || null,
+    city: city || null,
+    descrip: descrip || null,
+    iconUrl: iconUrl || null
+  };
+
+  currentWeatherData.push(pushObj);
+};
+
+var addFiveDayForecastData = function(data) {
+  fiveDayForecastData = [];
+  var sumTemp1 = 0;
+  var sumTemp2 = 0;
+  var sumTemp3 = 0;
+  var sumTemp4 = 0;
+  var sumTemp5 = 0;
+
+  for (var i = 0; i < data.list.length; i++) {
+    if (i < 8) {
+      sumTemp1 += data.list[i].main.temp; 
+    };
+    if (i > 7 && i < 16) {
+      sumTemp2 += data.list[i].main.temp; 
+    };
+    if (i > 15 && i < 24) {
+      sumTemp3 += data.list[i].main.temp; 
+    };
+    if (i > 23 && i < 32) {
+      sumTemp4 += data.list[i].main.temp; 
+    };
+    if (i > 31) {
+      sumTemp5 += data.list[i].main.temp; 
+    };
+  };
+
+  var obj1 = {
```

Comment:
Mapping every object individually works, but there are definitely more efficient/clean ways of doing it (using `.map` for example)